lib/Zonemaster/CLI.pm

```diff
@@ -112,6 +114,13 @@ has 'show_module' => (
     default       => 0,
 );
 
+has 'show_testcase' => (
+    is            => 'ro',
+    isa           => 'Bool',
+    documentation => __( 'Print the name of the test case (method) on entries.' ),
```

Comment:
I don't think we should refer to test cases as methods at the interface level. The interface documentation shouldn't leak implementation details such as how test cases are implemented in code.

Reply:
You are right, I modify that.